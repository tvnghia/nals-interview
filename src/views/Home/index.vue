<template>
  <p class="u-mt-40 headline--lg">Home page...</p>
</template>

<script>
export default {
  name: 'Home'
}
</script>
