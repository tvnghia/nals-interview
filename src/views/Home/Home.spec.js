import { shallowMount } from '@vue/test-utils'
import wrapperOps from '@/config-test/wrapper'
import Home from '@/views/Home'

describe('Home.vue', () => {
  const wrapper = shallowMount(Home, wrapperOps)

  describe('when Home was mounted', () => {
    it('should Home render correctly', () => {
      expect(wrapper.exists()).toBeTruthy()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
