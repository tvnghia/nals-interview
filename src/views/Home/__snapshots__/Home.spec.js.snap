// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue when Home was mounted should Home render correctly 1`] = `<p class="u-mt-40 headline--lg">Home page...</p>`;
